scripts/google.js

```diff
@@ -22,6 +22,7 @@ module.exports = (robot) => {
         return events.map(event => {
           return [
             '',
+            `> ${getDayName(event.date, 'pt-br')}`,
```

Comment:
Minor: To make it easier to support more locales in the future.

```suggestion
          const locale = 'pt-br';
          
          return [
            '',
            `> ${getDayName(event.date, 'pt-br')}`,
```

Reply:
Done!